src/Tribe/Capabilities.php

```diff
@@ -15,24 +47,64 @@ class Tribe__Events__Capabilities {
 			'delete_private_posts',
 			'delete_published_posts',
 			'publish_posts',
-		),
-		'author' => array( // full permissions for content the user created
+		],
+		//Full permissions for content the user created.
```

Comment:
:no_entry_sign: No space found before comment text; expected "// Full permissions for content the user created." but found "//Full permissions for content the user created."

<sub>hash: 44155f99a0df1fc744f009779b8c139b</sub>